numerical_methods/rungekutta.cpp

```diff
@@ -0,0 +1,133 @@
+/**
+ * @{
+ * \file
+ * \brief [Runge Kutta fourth order](https://en.wikipedia.org/wiki/Runge%E2%80%93Kutta_methods) method implementation
+ *
+ * \author [Rudra Prasad Das](http://github.com/rudra697)
+ *
+ * \details
+ * It solves the unknown value of y 
+ * for a given value of x
+ * only first oreder differential equations
+ * can be solved
+ * \example
+ * it solves dy/dx=(x-y)/2 at a 
+ * given x for given initial 
+ * conditions
+ * There can be many such equations 
+ * **/
+#include <iostream> /// for io operations
+#include <vector>   ///for using the vector container
+#include <cassert>  /// asserting the test functions
+
+/**
+ * @brief The change() function is used 
+ * to return the updated iterative value corresponding 
+ * to the given function
+ * @param x is the value corresponding to the x coordinate
+ * @param y is the value corresponding to the y coordinate 
+ * @returns the computed function value at that call
+ */
+static double change(double x, double y) 
```

Comment:
Shouldn't this function be inside the namespaces?

Reply:
the second reviewer suggested to keep it outside

Reply:
@Panquesito7